apps/examples/webserver/webserver_main.c

```diff
@@ -111,25 +112,27 @@ static int http_send_chunk(struct http_client_t *client)
 	memset(msg_body, 'a', MAX_DATA_SIZE);
 	msg_body[MAX_DATA_SIZE] = 0;
 
-	ret = http_send_response_chunk(client, 200, "OK", msg_body, NULL, FIRST_DATA);
+	ret = http_send_response_chunk(client, HTTP_200_RESPONSE, "OK", msg_body, MAX_DATA_SIZE, NULL, FIRST_DATA);
```

Comment:
Why is body_len always MAX_DATA_SIZE?
Then, parameter body_len of 'prepare_chunk_body' is also MAX_DATA_SIZE.
It can make overflow in memcpy

Reply:
MAX_DATA_SIZE is 50 bytes and HTTP_CONF_MAX_REQUEST_LENGTH is 4096 bytes